### PR TITLE
made changes on the views and added the form request. Do have a look

### PR DESCRIPTION
**Pull request by Faith-qa**

### CODE REVIEW

File: helprequest/views.py [hunk 1]
```diff
@@ -73,3 +86,24 @@ def form_valid(self, form):
...
 
+
+@method_decorator(login_required, name='dispatch')
+class HelpResponse(FormView):
+    template_name = 'help_response.html'
+    form_class = HelpResponseForm
+    print(form_class)
+    success_url = 'success'
+def gethelp(request):
+        if request.method == 'POST':
```
Comment (Emmastro) [inline helprequest/views.py]: Identation issue

File: helprequest/views.py [hunk 2]
```diff
@@ -73,3 +86,24 @@ def form_valid(self, form):
...
         return super().form_valid(form)
 
+
+@method_decorator(login_required, name='dispatch')
+class HelpResponse(FormView):
+    template_name = 'help_response.html'
+    form_class = HelpResponseForm
+    print(form_class)
+    success_url = 'success'
+def gethelp(request):
```
Comment (Emmastro) [inline helprequest/views.py]: What's the difference between this function and the HelpResponse class view?

File: helprequest/views.py [hunk 3]
```diff
@@ -73,3 +86,24 @@ def form_valid(self, form):
...
+        if request.method == 'POST':
+             if form.is_valid():
```
Comment (Emmastro) [inline helprequest/views.py]: where is the form variable declared here?

File: helprequest/views.py [hunk 4]
```diff
@@ -73,3 +86,24 @@ def form_valid(self, form):
...
+
+                form.save()
+                User = HelpRequest.medic
+                return redirect('home')
+        
+
+def form_valid(self, form):
```
Comment (Emmastro) [inline helprequest/views.py:87]: what is this function for?

File: helprequest/views.py [hunk 5]
```diff
@@ -47,9 +47,22 @@ class RequestHelp(FormView):
     """
     TODO: on save of the help request, link the user requesting help to the patient fild of the help request
     """
+
+
     template_name = 'help_request.html'
     form_class = HelpRequestForm
     success_url = 'status'
+    def gethelp(request):
```
Comment (Emmastro) [inline helprequest/views.py]: in which instance is this function going to be called?

**Review by Emmastro — CHANGES_REQUESTED** [review]
There are a few things that need to be reviewed before merging. Check them out.

File: helprequest/views.py (hunk 1 above)
Comment (Faith-qa) [reply helprequest/views.py]: corrected

File: helprequest/views.py (hunk 2 above)
Comment (Faith-qa) [reply helprequest/views.py]: come to think of it, no difference

File: helprequest/views.py (hunk 4 above)
Comment (Faith-qa) [reply helprequest/views.py:87]: were you not the one who created this?

**Pull request closed without merging**